server/routes/uploadProfileImage.js

```diff
@@ -0,0 +1,46 @@
+const express = require("express");
+const app = express();
+const multer = require("multer");
+const multerS3 = require("multer-s3");
+const AWS = require("aws-sdk");
+const fs = require("fs");
+
+const s3 = new AWS.S3({
+  accessKeyId: process.env.ACCESS_KEY_ID,
+  secretAccessKey: process.env.SECRET_ACCESS_KEY,
+  region: process.env.BUCKET_REGION,
+});
+
+const uploadS3 = multer({
+  storage: multerS3({
+    s3,
+    bucket: "teamruntimeterror",
+    key: (req, file, cb) => {
+      cb(null, Date.now().toString() + "-" + file.originalname);
+    },
+  }),
+});
+
+app.post("/upload", uploadS3.single("file"), (req, res) => {
+  console.log(req.file);
+});
+
+app.listen(process.env.PORT || 3009);
+
+// const uploadFile = (fileName) => {
```

Comment:
Please uncomment code here

Reply:
replaced the whole file with a new one which works on both conditions